sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -455,22 +455,25 @@ def _extract_data_cb(self, messages: Any) -> Tuple[str, List[QueueMessage]]:
 class QueueProperties(DictMixin):
     """Queue Properties.
 
-    :keyword str name:
+    :keyword Optional[str] name:
```

Comment:
Can we also update `_from_generated`?

Reply:
Hi Anna, maybe this is obvious (and it is going right over my head), but what is the change we are looking for here?

Here is the definition for `_from_generated`:
``` 
@classmethod
    def _from_generated(cls, generated: Any) -> Self:
        props = cls()
        props.name = generated.name
        props.metadata = generated.metadata
        return props
```

Do we want to omit setting `props.name`, or what type of update are we looking for here?